content/kotsadm/installing/automating.md

```diff
@@ -33,7 +34,16 @@ This file will be:
 ```yaml
 apiVersion: kots.io/v1beta1
 kind: ConfigValues
-...
+metadata:
+  name: myapp
+spec:
+  values:
+    app_name:
+      value: MyApp
+    gitlab_app_secret:
+      valuePlaintext: password123
+    primary_domain:
+      default: somebigbank.org
```

Comment:
That seems like a weird default. Should this be `value` instead?

Reply:
While you could, of course, include defaults in here, presumably the this config.yaml would be merged with the upstream so defaults would be filled in.

Reply:
This just shows that  `default` can be set here.  There's an example with `value` already.